protected/views/patient/_fuzzy_date.php

```diff
@@ -18,46 +18,53 @@
  */
 ?>
 <?php
+/**
+ * @var $form FormLayout
+ * @var $label string
+ * @var $empty bool
+ */
 $label = @$label ?: 'Date';
 $labelColumns = @$form ? $form->columns('label') : 'large-3 column';
 $fieldColumns = @$form ? $form->columns('field') : 'large-9 column end';
 if (isset($date)) {
     list($sel_year, $sel_month, $sel_day) = explode('-', $date);
-} else {
+} elseif (!isset($empty) || !$empty) {
```

Comment:
Apparently that doesn't work in this case, and also doesn't raise an inspection.